src/ApiService.php

```diff
@@ -356,6 +356,27 @@ public function findCustomerByRegistryIdentifier(string $registryIdentifier): ?C
 		return null;
 		}
 
+	/**
+	 * @param string $oldErpId
+	 * @return Customer|null
+	 * @throws ApiTransferException
+	 */
+	public function findCustomerByOldErpId(string $oldErpId): ?Customer
+		{
+		$filters = [[
+			'name'     => 'custentity_cus_crm_id',
```

Comment:
Out of curiosity - Zašto RSM pominje CRM u vezi s ovim ID, kada je reč o starom ERP (M&I)?